pre-docs/applying_manifest_blocks.md

```diff
@@ -308,4 +308,4 @@ gem must installed with the puppet agent on the proxy target must be at least
 version 1.8.1.
 
 
-
+[Bolt project directory]: ./bolt_project_directory.md
```

Comment:
I would recommend against putting links in task steps, as they tend to distract people and lead them away from what they're trying to accomplish. I can add a Related information link instead at the bottom of these two tasks.